### ISSUE TITLE
blackberry.identity docs missing required permissions info

### ISSUE THREAD

**Issue by astanley** [blackberry/WebWorks-API-Docs#375]
From the BlackBerry Dev experience review:

The documentation page for blackberry.identity does not indicate which required permission must be specific in config.xml

https://developer.blackberry.com/html5/apis/blackberry.identity.html

Reported issue:

*needs to specify that permit tags need to be inside permissions parent. also it says Playbook only is this needed for bb10 too? Example: 
    rim:permissions 
        rim:permit read_device_identifying_information /rim:permit 
    /rim:permissions

**Comment by mlantz** [comment]
I looked at that yesterday, and I noticed that problem with the Playbook only. Are all permissons required for BB10 too? If so I can change that in the template that all permissions are for both playbook and BB10 otherwise I'll have to add a note in the BB10 part.

**Comment by kwallis** [comment]
Yup, most/all permissions are carried over from PlayBook to BlackBerry 10.

**Comment by cdelcol** [comment]
I don't think it is a blanket statement, though. I suspect they will need to be duped, not just a change in the template.

Anyone? @jeffheifetz @nukulb

**Comment by mlantz** [comment]
@cdelcol do you mean that it should specifically state in its bubble on the page that it requires permissions to be set? Because I edited the template.
